exercises/nucleotide-count/nucleotide_count_test.go

```diff
@@ -34,20 +34,17 @@ var tallyTests = []struct {
 func TestNucleotideCounts(t *testing.T) {
 	for _, tt := range tallyTests {
 		dna := DNA(tt.strand)
-		count, _ := dna.Count(tt.nucleotide)
-		if count != tt.expected {
+		if count, err := dna.Count(tt.nucleotide); err != nil {
+			t.Fatal(err)
+		} else if count != tt.expected {
```

Comment:
will have to watch though, `count` might not be in scope since now `count, err :=` is in the `if`.